dnf-behave-tests/features/repo-sync.feature

```diff
@@ -101,4 +109,6 @@ Given I use repository "dnf-ci-fedora" with configuration
       | repo_gpgcheck | 1     |
  When I execute dnf with args "makecache"
  Then the exit code is 1
+  And stderr contains "Errors during downloading metadata for repository 'dnf-ci-fedora':"
+  And stderr contains "  - Curl error \(37\): Couldn't read a file:// file"
```

Comment:
I think that the only case where it makes sense is with the "All mirrors were tried" message and it could be printed only when this is the final error. But it's not an elegant solution either and there could be a situation where the missing message would be confusing, so I am not sure.

Reply:
Okay... in case you can distinguish the "All mirrors were tried" error by its type, you could use that for the decision whether to print the details. But quickly checking the code, I don't think it has a unique error type. I don't think we want to compare the strings for this, that's too ugly...